bridge/rd60xx_to_mqtt.py

```diff
@@ -279,10 +327,225 @@ async def _mqtt_outbound(self, identity:str, state:dict):
                 topic = f"{self._mqtt_base_topic}/psu/{identity}/state"
                 self._logger.debug("Msg leaving on topic: '%s', for identity: '%s' with payload: '%s'", topic, identity, repr(state))
                 await self._mqtt_client.publish(topic, payload=json.dumps(state))
+
+                # Update discovery with real firmware version if we haven't yet
+                if (self._mqtt_discovery_enabled and
+                    identity not in self._firmware_version_updated and
+                    'firmware_version' in state and
+                    identity in self._psus):
+
+                    psu = self._psus[identity]
+                    name = self._psu_identity_to_name.get(identity, "")
+                    self._logger.info("Updating device info with firmware version %s for %s", state['firmware_version'], identity)
+                    await self._publish_discovery_config(identity, psu.model, name, state['firmware_version'])
+                    self._firmware_version_updated.add(identity)
+
             except aiomqtt.MqttError:
                 # Ignore MQTT errors
                 pass
 
+    async def _publish_discovery_config(self, identity:str, model:int, name:str, firmware_version:str=None):
+        """Publish Home Assistant MQTT Discovery configuration for a PSU"""
+
+        if not self._mqtt_discovery_enabled or not self._mqtt_client:
+            return
+
+        self._logger.info("Publishing MQTT Discovery config for %s", identity)
+
+        # Extract current rating from model number
+        # Model format: VVCCR where VV=voltage, CC=current, R=revision
+        # Example: 60301 = 60V, 30A, revision 1
+        model_str = str(model)
+        current_rating = int(model_str[2:4])
+
+        # Set limits based on Riden PSU specifications
+        # All RD60xx models: 60V max output, 62V max OVP
+        # Current/OCP: nominal + 0.2A (e.g., 30A → 30.2A max)
```

Comment:
Not sure if this is right, mostly a guess

Reply:
I've got a few of them, the 6006, 6012P and 6018 - they all follow that general pattern. So far I'd say that was a pretty good assumption.